### PR TITLE
Add templates to run Cromwell

### PR DESCRIPTION
**Pull request by vladsavelyev**
Adding a repo with Cromwell config templates and run examples relevant to CPG.

### CODE REVIEW

File: cromwell.template.conf [hunk 1]
```diff
@@ -0,0 +1,147 @@
...
+
+        genomics {
+          # A reference to an auth defined in the `google` stanza at the top. This auth is used
+          # to create pipelines and manipulate auth JSONs.
+          auth = "application_default"
+
+          # Endpoint for APIs, no reason to change this unless directed by Google.
+          endpoint-url = "https://lifesciences.googleapis.com/"
+
+          # Currently Cloud Life Sciences API is available only in `us-central1` and
```
Comment (lgruen) [inline cromwell.template.conf]: It's a few more, maybe just put a link here?
https://cloud.google.com/life-sciences/docs/concepts/locations

File: cromwell.template.conf [hunk 2]
```diff
@@ -0,0 +1,147 @@
...
+          cpu: 1
+          failOnStderr: false
+          continueOnReturnCode: 0
+          memory: "2048 MB"
+          bootDiskSizeGb: 10
+          # Allowed to be a String, or a list of Strings
+          disks: "local-disk 10 SSD"
+          noAddress: false
+          preemptible: 0
+          zones: ["us-central1-a", "us-central1-b"]
```
Comment (lgruen) [inline cromwell.template.conf]: Can this be omitted? Otherwise maybe add a comment that it needs to match the above setting?

**Review by lgruen — APPROVED** [review]
Thanks a lot, this is great! Added a few minor comments.

File: cromwell.template.conf (hunk 2 above)
Comment (vladsavelyev) [reply cromwell.template.conf]: Yep, zones can be omitted!

**Pull request merged**